bridge/bridge.go

```diff
@@ -246,6 +248,21 @@ func (b *Bridge) add(containerId string, quiet bool) {
 	}
 }
 
+func mapTags(metadata map[string]string, container *dockerapi.Container) []string {
+	// execute go-template on tags using containger description in docker-client
+	// !! the go structure is used not the docker json description
+	var tags []string
+	var tag bytes.Buffer
+	var metadata_tags = metadata["tags"]
+	tagTemplate := template.Must(template.New("tags").Parse(metadata_tags))
+	err := tagTemplate.Execute(&tag, container)
+	if err != nil {
+		log.Println("executing template:", err)
+	}
+	tags = strings.Split(tag.String(), ",")
```

Comment:
`tag` could be nil here that would cause panic, since `tagTemplate.Execute` at line 258 could failed with err, in this case, it's better to return the original tags.